### PR TITLE
lightning: add doc for migrating parquet files

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

A followup of v6.3: revert this commit https://github.com/pingcap/docs-cn/pull/11261/commits/23c2684ba70ea3095569bbf88fa20f604488c9ed

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/11261

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-parquet-files-to-tidb.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: 从 Parquet 文件迁移数据到 TiDB
+summary: 介绍如何使用 TiDB Lightning 从 Parquet 文件迁移数据到 TiDB。
+---
+
+# 从 Parquet 文件迁移数据到 TiDB
+
+本文介绍如何使用 TiDB Lightning 从 Apache Parquet 文件迁移数据到 TiDB，以及如何生成 Parquet 文件。
+
+本文主要使用从 Apache Hive 中导出的 Parquet 文件进行数据迁移。如果你从 Amazon Aurora 中导出 Parquet 文件，请参照[从 Amazon Aurora 迁移数据到 TiDB](/migrate-aurora-to-tidb.md)。
```
Comment (Oreoxmt) [inline migrate-from-parquet-files-to-tidb.md:10]:
```suggestion
如果你从 Amazon Aurora 中导出 Parquet 文件，请参照[从 Amazon Aurora 迁移数据到 TiDB](/migrate-aurora-to-tidb.md)。
```

File: migrate-from-parquet-files-to-tidb.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+    执行上述语句后，表数据就成功导出到 HDFS 系统里。
+
+2. 使用 `hdfs -get` 命令将 Parquet 文件导出到本地：
+
+    ```shell
+    hdfs dfs -get /path/in/hdfs /path/in/local
+    ```
+
+    完成导出后，如果你需要将 HDFS 里导出的 Parquet 文件删除，可以直接将这个临时表删掉：
```
Comment (Oreoxmt) [inline migrate-from-parquet-files-to-tidb.md:38]: 这里删除“临时表”是 test 还是 temp

File: migrate-from-parquet-files-to-tidb.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+1. 在 Hive 中执行如下 SQL 语句：
+
+    ```sql
+    CREATE TABLE temp STORED AS PARQUET LOCATION '/path/in/hdfs'
+    AS SELECT * FROM test;
+    ```
+
+    执行上述语句后，表数据就成功导出到 HDFS 系统里。
+
+2. 使用 `hdfs -get` 命令将 Parquet 文件导出到本地：
```
Comment (Oreoxmt) [inline migrate-from-parquet-files-to-tidb.md:32]:
```suggestion
2. 使用 `hdfs dfs -get` 命令将 Parquet 文件导出到本地：
```

File: migrate-from-parquet-files-to-tidb.md (hunk 2 above)
Comment (buchuitoudegou) [reply migrate-from-parquet-files-to-tidb.md:38]: temp 是临时表

File: migrate-from-parquet-files-to-tidb.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+    2. 如果直接在命令行中启动程序，可能会因为 `SIGHUP` 信号而退出，建议配合 `nohup` 或 `screen` 等工具运行 `tidb-lightning`：
+
+        ```shell
+        nohup tiup tidb-lightning -config tidb-lightning.toml > nohup.out 2>&1 &
+        ```
+
+2. 导入开始后，可以采用以下任意方式查看进度：
+
+    - 通过 `grep` 日志关键字 `progress` 查看进度，默认 5 分钟更新一次。
```
[the remark this answers is not in the document]
Comment (buchuitoudegou) [reply migrate-from-parquet-files-to-tidb.md:115]: 感觉不太必要吧。而且这个描述是之前的文档一直在用的。

File: migrate-from-parquet-files-to-tidb.md (hunk 2 above)
Comment (Oreoxmt) [reply migrate-from-parquet-files-to-tidb.md:38]: 那下面删除的 SQL 是不是需要修改为 `DROP TABLE temp;`

File: migrate-from-parquet-files-to-tidb.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+2. 使用 `hdfs dfs -get` 命令将 Parquet 文件导出到本地：
+
+    ```shell
+    hdfs dfs -get /path/in/hdfs /path/in/local
+    ```
+
+    完成导出后，如果你需要将 HDFS 里导出的 Parquet 文件删除，可以直接将这个临时表删掉：
+
+    ```sql
+    DROP TABLE test;
```
Comment (buchuitoudegou) [inline migrate-from-parquet-files-to-tidb.md:41]:
```suggestion
    DROP TABLE temp;
```

File: migrate-from-parquet-files-to-tidb.md (hunk 2 above)
Comment (buchuitoudegou) [reply migrate-from-parquet-files-to-tidb.md:38]: 0957aa4

File: migrate-from-parquet-files-to-tidb.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+    执行上述语句后，表数据就成功导出到 HDFS 系统里。
+
+2. 使用 `hdfs dfs -get` 命令将 Parquet 文件导出到本地：
+
+    ```shell
+    hdfs dfs -get /path/in/hdfs /path/in/local
+    ```
+
+    完成导出后，如果你需要将 HDFS 里导出的 Parquet 文件删除，可以直接将这个临时表删掉：
```
Comment (Oreoxmt) [inline migrate-from-parquet-files-to-tidb.md:38]:
```suggestion
    完成导出后，如果你需要将 HDFS 里导出的 Parquet 文件删除，可以直接将这个临时表 (`temp`) 删掉：
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by buchuitoudegou — APPROVED** [review]

**Pull request merged**